README.md

```diff
@@ -30,19 +30,42 @@ server.plugin.register({
           }
          ]
       }],
-      mongo: require("mongodb")
+      mongo: require("mongodb"),
+      connectionOptions: {
+          server: {
+              poolSize: 15,
+              socketTimeout: 10000
+          },
+          replSet: {
+              connectWithNoPrimary: false
+          }
+      }
   }
 });
 
 ```
 
+`connectionOptions` parameter can be passed either as a child of options or per db. `connectionOptions` set for specific db take precedance over general ones.
```

Comment:
implementation wise it doesn't add complexity and seems like a nice feature (in our case we wouldn't want to repeat those for every db and would set the same settings for all of them :)